mlir/tools/miopen-gen/miopen-gen.cpp

```diff
@@ -1275,14 +1275,16 @@ createVerifierFunc(ModuleOp &module, const KernelIF &kernel,
     return b.create<arith::ConstantFloatOp>(loc, apVal, elemFType);
   };
 
-  mlir::Value fval00, fval000001, fval001;
+  mlir::Value fval00, fval000001, fval001, fval025;
+  float f16_threshold = 0.25f;
```

Comment:
Can we get away with `0.0001f`?

Reply:
which number are you referring to? Both 0.001f and 0.000001f are used but not 0.0001f.

Reply:
I'm saying the threshold shouldn't be 25% but some smaller number

... come to think it, while we're here, these long strings of 0s could use scientific notation.

But good to know